compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformTextInputMethodRequest.skiko.kt

```diff
@@ -16,4 +16,65 @@
 
 package androidx.compose.ui.platform
 
-actual interface PlatformTextInputMethodRequest
\ No newline at end of file
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.TextRange
+import androidx.compose.ui.text.input.EditCommand
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.ImeOptions
+import androidx.compose.ui.text.input.TextFieldValue
+
+actual interface PlatformTextInputMethodRequest {
+    val state: TextFieldStateAdapter
+    val imeOptions: ImeOptions
+    val onEditCommand: (List<EditCommand>) -> Unit
+    val onImeAction: ((ImeAction) -> Unit)?
+}
+
+/**
+ * The purpose of this interface is to provide, in the `ui` module, an adapter for
+ * `TransformedTextFieldStateAdapter`, which is in the `foundation` module.
+ *
+ * It exposes all the properties of `TransformedTextFieldStateAdapter` that are necessary for
+ * implementing input methods.
+ */
+interface TextFieldStateAdapter {
```

Comment:
Another stable public API

Reply:
The comment above it explains why it's needed.